### PR TITLE
Add First Page of my Portfolio

### PR DESCRIPTION
**Pull request by kiboma2021**
This is my first page of my portfolio as a software engineer

### CODE REVIEW

**Review by koomekevo — CHANGES_REQUESTED** [review]
Hi @kiboma2021, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Your README is descriptive.
:heavy_check_mark: Your project has no linter errors.

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly adhere to the design of the Figma template as shown in the below screenshot.

bencorrectdesign

- [ ] Below is a screenshot of your current design. 

benwrongdesign

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,73 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (ahmetbozaci) [inline README.md:73]: ### OPTIONAL
_I write it as an optional but this is a requirement that is not mentioned in the first review. Please fix it._

- [ ] Please fix the broken link in the license.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    />
+    <link rel="stylesheet" href="styles/main.css" />
+  </head>
+  <body>
+    <div class="container">
+      <main>
+        <nav>
+          <a href="#">Kiboma's</a
+          ><a><img src="imgs/showhide_buttons.png" alt="" /></a>
+        </nav>
```
Comment (ahmetbozaci) [inline index.html:24]: ### OPTIONAL
_I write it as an optional but this is a requirement that is not mentioned in the first review. Please fix it._


- [ ] This part should be wrapped by a header tag. Please fix it. 

- [ ] Only the navbar icon should be wrapped by the nav tag. You should move the tag outside the nav tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles/main.css" />
+  </head>
+  <body>
+    <div class="container">
+      <main>
```
Comment (ahmetbozaci) [inline index.html:20]: ### OPTIONAL
_I write it as an optional but this is a requirement that is not mentioned in the first review. Please fix it._


- [ ] You shouldn't wrap all body with div and main tag. Please delete the main and div tags.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+          me.
+        </p>
+        <h5>LET’S CONNECT</h5>
+        <ul class="flex">
+          <li><img src="imgs/Twitter.png" alt="Twitter Icon" /></li>
+          <li><img src="imgs/Linkedin-icon.png" alt="Linkedin-icon" /></li>
+          <li><img src="imgs/M.png" alt="M icon" /></li>
+          <li><img src="imgs/Github.png" alt="Github Icon" /></li>
+          <li><img src="imgs/hands.png" alt="hands" /></li>
+        </ul>
```
Comment (ahmetbozaci) [inline index.html:39]: ### Requirement
- [ ] Nice to fix the design.  You should add a background image like in the design.
- [ ] To stick to the design you should change the text color, and font size. 

### ORIGINAL
image
____

### YOUR DESIGN
image



____

### OPTIONAL
_I write it as an optional but this is a requirement that is not mentioned in the first review. Please fix it._


- [ ] This part should be wrapped by a section tag. Please fix it.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


@kiboma2021  ✋🏼 
#### Great work on making the changes requested by a previous reviewer 👏 
#### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
___
### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏

almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: README.md [hunk 5]
```diff
@@ -1 +1,73 @@
...
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://ben-microverse-project.com)
```
Comment (DeliceLydia) [inline README.md:13]: ## Optional
- [ ] I suggest that you fix your demo link

**Review by DeliceLydia — APPROVED** [review]
## Hi @kiboma2021, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Optional
- [ ] I suggest that you read about [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) to understand better how to structure your HTML page



### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**